agent/ae/ae_test.go

```diff
@@ -401,7 +402,6 @@ func (m *mock) SyncChanges() error {
 
 func testSyncer(t *testing.T) *StateSyncer {
 	logger := hclog.New(&hclog.LoggerOptions{
-		Level:  0,
```

Comment:
`0` is the zero value for this field, so no need to explicit set it to 0 here. I removed a couple instances of the zero in this PR while I was looking over all the places we set level in tests.